### DISCUSSION TITLE
NjRD

### DISCUSSION THREAD

**Discussion by SensoredRooster** [KnollFrank/HowBadIsMyBatch#7] [Ideas]
Hi.  I hope this comment finds you well.  I'm a permanently disabled full time U.S. employee ("citizen/slave").  I had no idea just how wonderful 37 years of fabulous, sorely missed, health could end literally overnight.  April 6th, & April 27th of 2021, was my 1st/2nd Pfizer dose.  June 4th, 2021, I awoke not knowing my family's lives would be changed forever.  I don't have to get into the whole story here, I can if you'd like but you can also read my story @react19.org.  Anyways, the whole reason for me reaching out, is to applaud your level of fight you're bringing to this war.  I don't know what your motivation is, nor would it make a difference no matter the reason, I just want to say thank you.  I'm part of a non-profit that brings awareness around the Covid19 warfare.  Our new website just went live, and we are progressively adding to it.   I've been sending people to your site for well over a year now checking batch#'s, symptoms, etc.  I am currently laying out a design to incorporate all adverse reporting I can get my hands on.  I have some projects I'm working on currently that I want to merge with your data.  It'll be so much easier and faster for me to achieve what I'm thinking.  (I'd be open to some help if you do that sort of thing, and I'll pay you for your time if required).  My end goal is to is to have a 100% Interactive AI Searchbot that will search the vaers files and form whatever is being asked by the user.  Ill take as little as advice if you feel like you can offer as well. Anyways, thank you again for what you're doing.

**Comment by KnollFrank** [comment]
Hi,

I am sorry about what happened to you. My motivation is to just provide a weekly summary of the VAERS database, warn about the harm these vaccines can cause, and to continue what Craig Paardekooper already started on his website howbadismybatch.com. You can freely use all the code of this project. If you have any questions regarding the code, please ask. I couldn't find your story on @react19.org, there are so many.

Regards, Frank
